app/src/gms/kotlin/com/hippo/ehviewer/cronet/CronetKt.kt

```diff
@@ -22,9 +21,6 @@ import splitties.init.appCtx
 
 val cronetHttpClient: CronetEngine = CronetEngine.Builder(appCtx).apply {
     enableBrotli(true)
-    val cache = (appCtx.cacheDir.toOkioPath() / "http_cache").toFile().apply { mkdirs() }
-    setStoragePath(cache.absolutePath)
-    enableHttpCache(CronetEngine.Builder.HTTP_CACHE_DISK_NO_HTTP, 100 * 1024)
```

Comment:
IIRC this is for QUIC.

Reply:
https://developer.android.com/develop/connectivity/cronet/reference/org/chromium/net/CronetEngine.Builder.html#public-cronetengine.builder-enablehttpcache-int-cachemode,-long-maxsize
Yeah we don't cache http data, just use it for QUIC server information.

Reply:
Correct me if I'm wrong,  HTTP_CACHE_DISK_NO_HTTP means we cache https data but not http data?

Reply:
No, it just caches QUIC hints.

Reply:
Didn't we use 100KB before? No reason to bump it to 1024.

Reply:
Aha, should be 1KB